app/controllers/repairs_controller.rb

```diff
@@ -2,86 +2,46 @@ class RepairsController < ApplicationController
   before_filter :authenticate_user!
   load_and_authorize_resource
 
-  # GET /repairs
-  # GET /repairs.json
   def index
     @repairs = Repair.all
-
-    respond_to do |format|
-      format.html # index.html.erb
-      format.json { render json: @repairs }
-    end
   end
 
-  # GET /repairs/1
-  # GET /repairs/1.json
   def show
-    @repair = Repair.find(params[:id])
-
-    respond_to do |format|
-      format.html # show.html.erb
-      format.json { render json: @repair }
-    end
+    @last_editor = last_editor(@repair)
+    @page_title = "Repair for #{@repair.item}"
```

Comment:
because you don't have a custom `to_s` on `Item` I'm surprised that this isn't `@repair.item.name`